tests/unit/cube/qb/test_cubeqb_errorvalidation.py

```diff
@@ -918,7 +1057,9 @@ def test_pivoted_validation_multiple_measure_columns():
     assert isinstance(error, PivotedShapeMeasureColumnsExistError)
 
 
-def test_pivoted_validation_no_measures_defined_error():
+def test_pivoted_validation_no_measures_defined_error_codelist_true(
```

Comment:
You went overboard on this. Our testing is that no measure is failing. Delete the duplicates tests.